website/docs: Add steps to troubleshoot /initial-setup/ (#15011)
goauthentik/authentik: website/docs/troubleshooting/login.md

```

This will output a link, that can be used to instantly gain access to authentik as the user specified above. The link is valid for amount of years specified above, in this case, 10 years.

> ## Can't access initial setup

Reviewer: @melizeche am I understanding correctly that this troubleshooting tip is only for the first attempt, during installation, to access the `initial-setup` flow?

If yes, the above suggestion for the title is more descriptive. Also, we will need to add a parallel title for the other, original section, maybe something like `##Use recovery key to log in`... up above line 5.

> If you're unable to access the initial setup flow (`/if/flow/initial-setup/`), first try restarting the containers as this often resolves temporary issues.

Reviewer: ```suggestion
If you're unable to access the initial setup flow (`/if/flow/initial-setup/`), first try restarting the containers because this often resolves temporary issues.
```

> However, if the issue persists after restarting, you can directly reset the admin password using the following commands:

Reviewer: Wouldn't it be "set" and not "reset" because they never got to set it in the first place?

> Docker Compose deployments:

Author: I don't think that it applies it here, docker and kubectl should be installed system wide

    ```bash
    docker compose exec server ak changepassword akadmin
    ```

Kubernetes deployments:

    ```bash
    kubectl exec -it deployment/authentik-server -c server -- ak changepassword akadmin
    ```

> Follow the prompts to set a new password. You can then login via: `https://authentik.company/if/flow/default-authentication-flow/?next=%2F`

Reviewer 2: ```suggestion
After following the prompts to set a new password, you can then login via: `https://authentik.company/if/flow/default-authentication-flow/?next=%2F`
```

> Update the admin's email in the settings at `/if/user/#/settings`

Reviewer 2: ```suggestion
After logging in, you can set the email address and other settings for the account by navigating to **Directory** > **Users** and editing the user account.
```

:::note

> This method bypasses the initial setup flow and should only be used as a last resort. The initial setup flow is the recommended method to configure the admin user.

Reviewer: ```suggestion
This method bypasses the initial setup flow and should only be used as a last resort. The initial setup flow is the recommended method to configure the administrator user.
```

```

This will output a link, that can be used to instantly gain access to authentik as the user specified above. The link is valid for amount of years specified above, in this case, 10 years.

> ## Can't access Initial Setup

Reviewer 2: ```suggestion
## Can't access initial setup
```

> If you're unable to access the initial setup flow (`/if/flow/initial-setup/`), first try restarting the containers as this often resolves temporary issues:

Reviewer 2: ```suggestion
If you're unable to access the initial setup flow (`/if/flow/initial-setup/`), first try restarting the containers as this often resolves temporary issues.
```

> If the issue persists after restarting, you can reset the admin password directly through the container as a last resort:

Reviewer 2: ```suggestion
However, if the issue persists after restarting, you can directly reset the admin password using the following commands:
```

1. Access the server and reset the `akadmin` user's password:

>     For Docker Compose:

Reviewer 2: ```suggestion
Docker Compose deployments:
```

    ```bash
    docker compose exec server ak changepassword akadmin
    ```

>     For Kubernetes:

Reviewer 2: ```suggestion
Kubernetes deployments:
```

    ```bash
    kubectl exec -it deployment/authentik-server -c server -- ak changepassword akadmin
    ```

>     For CLI:
> 
>     ```bash
>     uv run ak changepassword akadmin
>     ```
> 

Reviewer 2: ```suggestion
```

Because this is customer facing, I wouldn't mention CLI since it's not a supported deployment method. Maybe this could be included in developer specific troubleshooting?

>     Follow the prompts to set a new password.

Reviewer 2: ```suggestion
Follow the prompts to set a new password. You can then login via: `https://authentik.company/if/flow/default-authentication-flow/?next=%2F`
```

> 
> 2. Log in using the admin interface:
> 
>     - URL: `https://your-authentik-url/if/flow/default-authentication-flow/?next=%2F`
>     - Username: `akadmin`
>     - Password: [the password you just set]
> 
> 3. (Optional) Update the admin's email in the settings at `/if/user/#/settings` if needed.
> 

Reviewer 2: ```suggestion
```
